Cargo.toml

```diff
@@ -36,6 +36,7 @@ procfs = "0.7.4"
 
 [dev-dependencies]
 insta = "0.12.0"
+pnet = "0.23.0"
```

Comment:
Why do we need this one as a dev-dependency? Wouldn't this imply that we build pnet twice?

Reply:
Yeah, but just for the tests. They use packet_builder, which has pnet as a transitive dependency. Since we're using some stuff from pnet in the tests, this wrecks havoc and I thought this would be the easiest solution. :)